{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "312f5476_eb7bb918",
        "filename": "src/main/java/com/googlesource/gerrit/plugins/multisite/forwarder/ForwardedIndexingHandlerWithRetries.java",
        "patchSetId": 3
      },
      "lineNbr": 118,
      "author": {
        "id": 1010134
      },
      "writtenOn": "2024-02-12T14:58:59Z",
      "side": 1,
      "message": "nit: when we follow the execution path starting from line 91, when task cancellation fails (`task.cancel(true)` returns false, we\u0027ll first get log statement:\n\n_Cannot cancel task_\n\nthen:\n\n_Cancelling the previous task for index ..._\n\nWhich doesn\u0027t make sense. Shall we return `false` when the task was not cancelled, to prevent printing a long statement in line 92?\n\nAlso, should we assume that the task was cancelled, when it was not found in the map?",
      "revId": "db37225f073d22e0c029c7ccfa8c3b54df428cda",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}